subcommands/keys/rotate_root.go

```diff
@@ -91,6 +96,11 @@ func doRotateRoot(cmd *cobra.Command, args []string) {
 
 	newid, newPk, newCreds := swapRootKey(root, curid, creds)
 	fmt.Println("= New root:", newid)
+	root.Signed.Reason = &client.RootChangeReason{
+		PolisId:   user.PolisId,
+		Message:   changeReason,
```

Comment:
Maybe this is just my phobia, but passing the user input `changeReason` to the signer and then transferring it as a payload of http request to the OTA server looks a bit scary to me :).

Reply:
It's a REST API that anyone can call. We have to protect at the API level or its just security theater. Your concern is valid but that becomes a "I want to review our backend" topic.

Reply:
My concern is that a malicious user may input huge value that has to be transferred and accepted by our backend (and maybe copied a few times before it arrives to the handler) before it's checked by the handler.

But, you're right, such user may just do the same directly via the rest api.

Reply:
still - doing monthly-ish reviews of nginx proxy settings is always a good idea